src/product/dashboard.njk

```diff
@@ -14,8 +14,10 @@ meta:
             <h4 class="font-light mt-2 text-gray-500">
                 Build visually appealing and useful UIs in minutes!
             </h4>
-            <div class="justify-center items-center m-auto w-full flex mt-10">
-                {% image "../images/dashboard/dashboard.svg", "FlowFuse Dashboard", [1920] %}
+            <div class="justify-center items-center m-auto w-full flex mt-10 gap-3 dashboard-image">
```

Comment:
I think on mobile, we should place the two smaller images next to each other, with the third one below them. What do you think?

Reply:
![image](https://github.com/user-attachments/assets/93c5b0f0-a9f5-4004-8ae4-7b3d1b4cc737)

same like this?

Reply:
Yes!